src/main/java/org/websoso/WSSServer/controller/UserController.java

```diff
@@ -90,6 +92,17 @@ public ResponseEntity<MyProfileResponse> getMyProfileInfo(Principal principal) {
                 .body(userService.getMyProfileInfo(user));
     }
 
+    @GetMapping("/profile/{userId}")
+    public ResponseEntity<ProfileGetResponse> getProfileInfo(Principal principal,
+                                                             @PathVariable("userId") Long userId) {
+        User user = principal == null ?
+                null :
+                userService.getUserOrException(Long.valueOf(principal.getName()));
```

Comment:
p2; [전에 남겨주신 글](https://medium.com/hayanmind-tech-blog-kr/3%ED%95%AD-%EC%97%B0%EC%82%B0%EC%9E%90%EC%9D%98-%EC%9E%AC%EB%B0%9C%EA%B2%AC-a3f815bcdbb8)에서 보이는 것과 다르게 삼항연산자를 개행하셨는데, 이렇게 하신 이유가 있을까요? 전 삼항연산자 개행에서 위 글을 따라서 통일하면 좋을 것 같아서요!
```suggestion
        User user = principal == null
                ? null
                : userService.getUserOrException(Long.valueOf(principal.getName()));
```

Reply:
오 개행을 잘못 했네요
저도 아직 익숙치 않은 것 같습니다 ㅋㅋ,, 수정하겠습니다!

Reply:
```suggestion
        User user = 
                principal == null
                ? null
                : userService.getUserOrException(Long.valueOf(principal.getName()));
```

어라 근데 다시 확인해보니까 이렇게 쓰라고 하는데... 어떻게 할까요 ㅋㅋ
제대로 안정해진 것 같아요

Reply:
저희 정한대로 윗부분처럼 수정하면 될 것 같숨니다!

Reply:
반영 완료!